compiler/x/codegen/OMRInstOpCode.hpp

```diff
@@ -511,6 +523,12 @@ class InstOpCode: public OMR::InstOpCode
                   supported = target->supportsFeature(OMR_FEATURE_X86_AVX512DQ);
                if (supported && (flags & X86FeatureProp_AVX512CD))
                   supported = target->supportsFeature(OMR_FEATURE_X86_AVX512CD);
+               if (supported && flags & X86FeatureProp_AVX512VBMI2)
```

Comment:
Can you add parentheses to these expressions so the reader doesn't have to reconcile the operator association rules in their head?